docs/terminology.md

```diff
@@ -10,17 +12,42 @@ the user can manage a group more easily. Having this given, a group can be
 treated like a single unit. All actions one can apply to a unit using
 `fleetctl`, can also be applied towards a group using `inagoctl`.
 
+There are two types of groups: slice groups and instance groups. Unit files,
+which contain an `@` sign are considered sliceable. If the unit files does **not**
+contain an `@` sign the unit is recognized an instance unit.
+
+Examples:
+
+### Sliceable group
+
+```nohighlight
+└── mygroup
+    ├── mygroup-bar@.service
+    └── mygroup-foo@.service
+```
+
+### Instance group
+
+```nohighlight
+└── timer
```

Comment:
sorry, trying to clear that up: timer is a bad example for instance group as we don't support timers yet. But that is an issue with the `.timer` suffix, not the concept of instance groups.
k8s is a better example as it is an instance group but does not contain timer:
```
├── k8s
│   ├── k8s-master
│   │   ├── k8s-master-apiserver.service
│   │   ├── k8s-master-controller-manager.service
│   │   ├── k8s-master-register.service
│   │   └── k8s-master-scheduler.service
│   └── k8s-node
│       ├── k8s-node-kubelet.service
│       └── k8s-node-proxy.service
```